package.json

```diff
@@ -218,17 +218,17 @@
     "node-unzip-2": "^0.2.8",
     "oclif": "^1.15.2",
     "open": "^7.0.2",
-    "patch-package": "6.1.2",
```

Comment:
Nope! You had already updated this about a month ago, it introduced build errors and I pinned it back:
* https://github.com/balena-io/balena-cli/pull/1608/commits/05d478b759a79e7e3f0ae9c1d4e5e9adce246409
* https://github.com/ds300/patch-package/pull/202
* https://github.com/ds300/patch-package/issues/201

I created a PR with a fix upstream (link above), the maintainer thought the solution should be different so the PR was closed; they said they'd do it themselves in a week, then they didn't. I created an issue in their repo (also above).

Reply:
I've pinned it but I think it's probably worth trying to remove those patches if at all possible since modifying into other packages can break expectations and is generally fragile (albeit I can also see the usefulness for short periods)